<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simulation</class>
 <widget class="QDialog" name="simulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1484</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>等效设备</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,3" columnminimumwidth="1,3">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="simTreeWidget">
          <column>
           <property name="text">
            <string>分系统</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTableWidget" name="simTableWidget">
          <column>
           <property name="text">
            <string>测试名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>创建时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>最近打开时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>备注</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>UUID</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>仿真表</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTreeWidget" name="treeWidget_3">
        <column>
         <property name="text">
          <string>等效设备</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>流程名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>record_uuid</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitSim">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
